wallet/src/de/schildbach/wallet/ui/CreateUsernameActivity.kt

```diff
@@ -229,4 +299,33 @@ class CreateUsernameActivity : InteractionAwareActivity(), TextWatcher {
     override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
     }
 
+//    private fun handleDecryptPIN(password: String) {
```

Comment:
Will we need this?

Reply:
we won't need this. the code was "moved" to the service

Reply:
now this code has been removed

Reply:
removed:  https://github.com/dashevo/dash-wallet/pull/393/commits/1df45863dd61ec1ce62105038f5247f9243a14a4